src/app/components/select-language/select-language.component.html

```diff
@@ -1,13 +1,20 @@
-<ion-select
-  *ngrxLet="currentLanguageData$ as currentLanguageData"
-  class="tw-w-fit"
-  interface="action-sheet"
-  [value]="currentLanguageData.code"
-  [cancelText]="'button.cancel' | translate"
-  (ionChange)="handleLanguageChange($event)">
-  <span class="fi fi-{{ currentLanguageData.flagIconCode }}" slot="label"></span>
+<ion-item class="tw-relative" *ngrxLet="currentLanguageData$ as currentLanguageData">
+  <ion-select
+    class="tw-w-fit tw-py-3"
+    interface="action-sheet"
+    justify="start"
+    [value]="currentLanguageData.code"
+    [cancelText]="'button.cancel' | translate"
+    (ionChange)="handleLanguageChange($event)">
+    <span class="tw-text-xs tw-font-bold" translate="select_language.label" slot="label"></span>
 
-  @for (language of supporedLanguages; track language) {
-    <ion-select-option [value]="language.code">{{ language.displayValue }}</ion-select-option>
-  }
-</ion-select>
+    @for (language of supporedLanguages; track language) {
+      <ion-select-option [value]="language.code">{{ language.displayValue }}</ion-select-option>
+    }
+  </ion-select>
+
+  <div class="tw-absolute tw-right-0">
```

Comment:
it will be nice to avoid using absolute in that case, it looks like we can build layout not braking layers order. You can also try to use 'label' of ion-select component

Reply:
It will be also nice to make sure, that component displayed correctly, without layout issues in public hader component